### PR TITLE
Include integration development link

### PR DESCRIPTION
**Pull request by seanvictory**
## Proposed change
Includes a link to the some-what hidden integration development page. Developers that don't find their favorite integrations listed are encouraged to contribute to Open Source development and integrate those systems into Home Assistant.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
The link serves as a call-to-action for community members that are interested in contributing but don't know where to start. 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/integrations/index.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
 {%- assign categories = components | map: 'ha_category' | join: ',' | join: ',' | split: ',' | uniq | sort -%}
 
 <p class='note'>
-  Support for these integrations is provided by the Home Assistant community.
+  Support for these integrations is provided by the Home Assistant community. Looking to contribute? Develop a <a href="https://developers.home-assistant.io/docs/creating_component_index">Home Assistant integration</a>.
```
Comment (frenck) [inline source/integrations/index.html:29]: Please use Markdown to link it.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/integrations/index.html (hunk 1 above)
Comment (seanvictory) [reply source/integrations/index.html:29]: @frenck Fixed. Thank you for your help!

**Comment by frenck** [comment]
First of all; Sorry, I was wrong with the HTML/Markdown bit, as this is not a normal page. So that is my bad.

Secondly, looking at the result, this will add quite a bit of text there making the header go multiline. I think that doesn't look good.

Especially considering we don't target developers on that page, but users.

image

Please note, that even with a working URL, it will go split across multiple lines.

I wonder if we should merge this at all...

**Comment by seanvictory** [comment]
What are your thoughts on changing it to just “Looking to contribute?” and linking the page on the word contribute?  That way it’s still all on one line.

Ultimately it’s okay if it can’t go right there. We can probably find someplace else to put similar wording.

image

**Comment by seanvictory** [comment]
What are your thoughts on `Interested in developing an integration?` and linking the page on `integration`?

**Comment by frenck** [comment]
Well, I think we should not put it here. We are trying to make our documentation more user-focussed.

We have a developer link in the footer of each page and have some more links on pages like the "Help" section.

I think we should not add this, as it is not user-focussed.

**Pull request closed without merging**